src/graph/graph.c

```diff
@@ -521,7 +433,7 @@ int Graph_AddRelationMatrix(Graph *g) {
         g->_relations = realloc(g->_relations, g->relation_cap * sizeof(GrB_Matrix));
     }
 
-    GrB_Matrix_new(&g->_relations[g->relation_count++], GrB_BOOL, g->node_cap, g->node_cap);
+    GrB_Matrix_new(&g->_relations[g->relation_count++], GrB_BOOL, _Graph_NodeCap(g), _Graph_NodeCap(g));
```

Comment:
`_Graph_Resize_Matrix` will shrink matrices to NodeCount before they get populated, so that would probably be a better allocation size.

Reply:
Same remark as for previous comment.